src/components/mgt-tasks/mgt-tasks.ts

```diff
@@ -833,89 +888,204 @@ export class MgtTasks extends MgtBaseComponent {
 
     return html`
       <div class="Task NewTask Incomplete">
-        <div class="InnerTask">
-          <span class="TaskHeader">
-            ${taskTitle}
-          </span>
-          <hr />
-          <span class="TaskDetails">
-            ${taskDresser} ${taskDrawer} ${taskPeople} ${taskDue}
-          </span>
+        <div class="TaskContent">
+          <div class="TaskDetailsContainer">
+            <div class="TaskTitle">
+              ${taskTitle}
+            </div>
+            <hr />
+            <div class="TaskDetails">
+              ${group} ${taskFolder} ${taskPeople} ${taskDue}
+            </div>
+          </div>
         </div>
         ${taskAdd}
       </div>
     `;
   }
 
-  private renderTaskHtml(task: ITask) {
+  private showPeoplePicker(task: ITask) {
+    if (this._showPeoplePicker) {
+      const isCurrentTask = task === this._currentTask;
+      if (isCurrentTask) {
+        return;
+      } else {
+        this.hidePeoplePicker();
+      }
+    }
+    this._currentTask = task;
+    this._showPeoplePicker = true;
+
+    // logic for already created tasks
+    if (this.renderRoot) {
+      // if shadowroot exists search for the task's assigned People and push to picker
+      const picker = this.getPeoplePicker(task);
+      const mgtPeople = this.getMgtPeople(task) as MgtPeople;
+
+      const assignedPeople: any = mgtPeople.people.map(person => {
+        return person.id;
+      });
+
+      if (picker) {
+        picker.selectUsersById(assignedPeople);
+      }
+    }
+  }
+
+  private hidePeoplePicker() {
+    const picker = this.getPeoplePicker(this._currentTask);
+    const mgtPeople = this.getMgtPeople(this._currentTask) as MgtPeople;
```

Comment:
same as above, no  need to case to MgtPeople

Reply:
Removed: 
76741d3